### ISSUE TITLE
[Controller Request]: Gamesir X2 Bluetooth

### ISSUE THREAD

**Issue by ghost** [ndeadly/MissionControl#592]
### Controller Name

Gamesir X2 Bluetooth Support

### Current Controller Behaviour

Controller is not successful in pairing with the console

### Supported Features

- [ ] Rumble/Vibration
- [ ] Motion Controls
- [ ] Player Number Indicator
- [ ] Battery Level Reporting

### Additional Details

I can provide any details if needed. unsure how to give the needed information as its been years since the last time i did this.

**Comment by ndeadly** [comment]
Are you able to use other controllers successfully? If the controller doesn't pair with the console at all, then it may be using Bluetooth LE. You can check this in Windows, for example, under the device properties in device manager.

**Comment by AlexS2412** [comment]
yes controllers like DS4 work just fine, i had a long session the other way with one. i was just curious about using this controller since it can attach to the switch and doesn't drift like my joycons haha

**Comment by Masamune3210** [comment]
controller page says it supports bluetooth 4.2, has no mention of LE, but that hasn't stopped others before from having it and just not noting it

**Comment by ndeadly** [comment]
> yes controllers like DS4 work just fine, i had a long session the other way with one. i was just curious about using this controller since it can attach to the switch and doesn't drift like my joycons haha

Then I would guess it's probably a BLE controller. If you can confirm this I can mark this request as such so I can revisit it when LE is working. If it's not an LE controller then we can investigate further why it's not connecting.

> controller page says it supports bluetooth 4.2, has no mention of LE, but that hasn't stopped others before from having it and just not noting it

In my experience, most controllers don't explicitly state they're using BLE, and stating the bluetooth version like that often implies they are.

**Comment by OfficialLayton** [comment]
Bump for this

**Comment by ndeadly** [comment]
@OfficialLayton please refer to the earlier comments regarding obtaining info about the controller
